### PR TITLE
Api tests

### PR DESCRIPTION
**Pull request by Corn1344**
Added docker network
Both containers run on the same network
Added API tests
Added script to stop flask application and docker-compose down so that docker-compose up is able to be run again.

### CODE REVIEW

File: fixdocker.sh
Comment (lundgren98) [inline fixdocker.sh:1]: najs

File: Jenkinsfile
Comment (lundgren98) [inline Jenkinsfile:1]: lite fult med tabs dock

**Review by lundgren98 — APPROVED** [review]
LGTM

**Review by davidhasenson — APPROVED** [review]
ok

**Pull request merged**